README.md

```diff
@@ -9,6 +9,11 @@ Use [Apollo Client](https://github.com/apollographql/apollo-client) as React
 
 `npm install react-apollo-hooks`
 
+If using yarn
```

Comment:
```suggestion
Or if using [yarn](https://yarnpkg.com/en/)
```

I think `Or` here could be helpful for people not experienced too much in JS tools.